databuilder/models/table_elasticsearch_document.py

```diff
@@ -26,6 +26,7 @@ def __init__(self,
         self.cluster = cluster
         self.schema_name = schema_name
         self.name = name
+        self.display_name = '{schema}.{table}'.format(schema=schema_name, table=name)
```

Comment:
i wonder why we don't make the display_name as an optional field with default value empty string. Then it provides the flexibility to allow different user to set different display_name(depends on the databuilder job input) instead of fixing it into schema.table?

Reply:
Yep, that's better. Just updated! @feng-tao